content/300-guides/500-other/880-troubleshooting-orm/100-help-articles/600-vercel-caching-issue.mdx

```diff
@@ -0,0 +1,57 @@
+---
+title: 'Vercel deployment caching workaround'
+metaTitle: 'Learn to configure your build process on Vercel to avoid caching-related problems'
+metaDescription: 'Learn to configure your build process on Vercel to avoid caching-related problems'
+---
+
+## Problem
+
+If you deploy an application using Prisma to [Vercel](https://vercel.com/), you may run into strange issues as you iterate on your Prisma schema and deploy updates to your app.
```

Comment:
```suggestion
If you deploy an application using Prisma to [Vercel](https://vercel.com/), you may run into query validation errors as you iterate on your Prisma schema and deploy updates to your app, although these queries perfectly work locally.
```

Reply:
(For older Prisma version we could include something like this, but I am not sure this is worth it and make the article a lot more complex by doing so)

Reply:
Can you provide Sabin with a up to date example of the error message @millsp ?
Maybe even a screenshot what it looks like in the logs?

Reply:
```
Prisma has detected that this project was built on Vercel, which caches dependencies. This leads to an outdated Prisma Client because Prisma's auto-generation isn't triggered. To fix this, make sure to run the \`prisma generate\` command during the build process.

Learn how: https://pris.ly/d/vercel-build
```